packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -711,6 +744,31 @@ function useActionState<S, P>(
   return [state, (payload: P) => {}, false];
 }
 
+function useHostTransitionStatus(): TransitionStatus {
+  const status = readContext<TransitionStatus>(
+    // $FlowFixMe[prop-missing] `readContext` only needs _currentValue
+    ({
+      // $FlowFixMe[incompatible-cast] TODO: Incorrect bottom value without access to Fiber config.
+      _currentValue: null,
+    }: ReactContext<TransitionStatus>),
+  );
+
+  hookLog.push({
+    displayName: null,
+    primitive: 'HostTransitionStatus',
+    stackError: new Error(),
+    value: status,
+    debugInfo: null,
+    dispatcherHookName: 'HostTransitionStatus',
+    wrapperNames: [
+      // react-dom
+      'FormStatus',
+    ],
```

Comment:
I wonder if we actually need to support renderer-specific hooks. Same as for hooks from core, we relying on the hook name, but at least we have one core. Here, I am assuming, if we had React Native renderer, we would also treat a hook named `useFormStatus` as primitive React hook.

Reply:
I can double check but I don't think so since we only look one frame above the dispatcher which is always a core hook. So even if you would have a custom `useFormStatus` hook, the stacktrace would consist of
```
dispatcher.useSomeDispatcherHook
React.useSomeReactHook
useFormstatus
```

Even if RN would implement have a `useFormStatus` hook that's not wrapping `useHostTransitionStatus`, we wouldn't mix them up with this implementation since we only expect `useFormStatus` as a wrapper of `useHostTransitionStatus`.

The whole `wrapperNames` concept seems unnecessary though in hindsight. If we keep the assumption that a dispatcher method is wrapped by a hook and no intermediate wrappers, we could just immediately use the wrapper name.

Reply:
Added a test to check we don't confuse hooks. @hoxyq let me know if that was the scenario you were talking about.

Also removed `wrapperNames` since we don't need that concept.